schema/index.js

```diff
@@ -1,9 +1,17 @@
 const { loginSchema } = require("./loginSchema");
 const { registerSchema } = require("./registerSchema");
 const { updateUserSchema } = require("./updateUserSchema");
+const { projectIdSchema, commentIdSchema, uuidSchema } = require("./params");
+const { projectSchema, projectUpdateSchema, typeIdSchema } = require("./projectSchema");
 
 module.exports = {
   loginSchema,
   registerSchema,
   updateUserSchema,
```

Comment:
Might as well hehe..
```suggestion
  updateUserSchema,
  createCommentSchema,
  updateCommentSchema,
```